lib/new_relic/agent/instrumentation/roda/roda_transaction_namer.rb

```diff
@@ -0,0 +1,35 @@
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+# frozen_string_literal: true
+
+module NewRelic
+  module Agent
+    module Instrumentation
+      module Roda
+        module TransactionNamer
+          extend self
+
+          ROOT = '/'.freeze
```

Comment:
We have similar constants elsewhere in the codebase set to `'/'`. We could take on a refactoring project to consolidate and move them into the New Relic constants file.

Reply:
I love that idea. I'll make a ticket for it!